### PR TITLE
Add example checking measurement free 9-qubit surface code state prep

### LINKED ISSUE #11: Test measurement-free state preparation gadget
As a way to further validate the tool, add a test of the measurement-free fault-tolerant logical zero-state encoding of the distance-three nine-qubit surface code in a one-dimensional qubit array in https://arxiv.org/pdf/2303.17211.

### PR DESCRIPTION
**Pull request by bachase**
Checks the state preparation gadget in arXiv:2303.17211

Resolves #11

### CODE REVIEW

**Comment by bachase** [comment]
@jordandsullivan and @natestemen -- tagged you on this but consider it optional. More an FYI given the earlier discussion on this

File: src/ucc_ft/QuantumSE.jl/src/SymbolicExecution.jl [hunk 1]
```diff
@@ -39,19 +39,11 @@ Base.copy(cfg::SymConfig) = SymConfig(cfg)
...
-end
-
 function CEvalHack(σ::CState, e)
-    @debugprintln("CEval: σ = ", σ)
-    @debugprintln("CEval: original expression = ", e)
+    println("CEval: σ = ", σ)
+    println("CEval: original expression = ", e)
     transformed = postwalk(x -> x isa Symbol ? x in keys(σ) ? σ[x] : x : x, e)
-    @debugprintln("CEval: transformed expression = ", transformed)
+    println("CEval: transformed expression = ", transformed)
```
[the remark this answers is not in the document]
Comment (bachase) [reply src/ucc_ft/QuantumSE.jl/src/SymbolicExecution.jl:46]: This was intentional for now as the macro was hard to manage in the workflows. Can revisit this later
